src/geocat/viz/taylor/taylor.py

```diff
@@ -0,0 +1,600 @@
+
+        # format each label with 2 decimal places
+        format_string = list(map(lambda x: "{0:0.2f}".format(x), stdLevel))
+        index = np.where(stdLevel == self.refstd)[0][0]
+        format_string[index] = label
+        tf2 = gf.DictFormatter(dict(list(zip(stdLevel, format_string))))
+
+        # Use customized GridHelperCurveLinear to define curved axis
+        ghelper = fa.GridHelperCurveLinear(
+            tr,
+            extremes=(
+                0,
+                np.pi / 2,  # 1st quadrant only
+                self.smin,
+                self.smax),
+            grid_locator1=gl1,
+            tick_formatter1=tf1,
+            grid_locator2=gl2,
+            tick_formatter2=tf2)
+
+        # Create graphical axes
+        ax = fa.FloatingSubplot(self.fig, rect, grid_helper=ghelper)
+        self.fig.add_subplot(ax)
+
+        # Adjust axes for Correlation
+        ax.axis["top"].set_axis_direction("bottom")  # "Angle axis"
+        ax.axis["top"].toggle(ticklabels=True, label=True)
+        ax.axis["top"].major_ticklabels.set_axis_direction("right")
+        ax.axis["top"].label.set_axis_direction("top")
+        ax.axis["top"].label.set_text("Correlation")
+
+        # Standard deviation ("X axis")
+        ax.axis["left"].set_axis_direction("bottom")
+
+        # Standard deviation ("Y axis")
+        ax.axis["right"].set_axis_direction("top")
+        ax.axis["right"].toggle(ticklabels=True, label=True)
+        ax.axis["right"].major_ticklabels.set_axis_direction("left")
+        ax.axis["right"].label.set_text("Standard deviation (Normalized)")
+
+        # Set font sizes, tick sizes, and padding
+        ax.axis['top', 'right'].label.set_fontsize(18)
+        ax.axis['top', 'right', 'left'].major_ticklabels.set_fontsize(16)
+        ax.axis['top', 'right', 'left'].major_ticks.set_ticksize(10)
+        ax.axis['top', 'right', 'left'].major_ticklabels.set_pad(8)
+        ax.axis['top', 'right'].label.set_pad(6)
+
+        # Bottom axis is not needed
+        ax.axis["bottom"].set_visible(False)
+
+        self._ax = ax  # Graphical axes
+        self.ax = ax.get_aux_axes(tr)  # Polar coordinates
+
+        # Add reference point stddev contour
+        t = np.linspace(0, np.pi / 2)
+        r = np.zeros_like(t) + self.refstd
+        h, = self.ax.plot(t,
+                          r,
+                          linewidth=1,
+                          linestyle=(0, (9, 5)),
+                          color='black',
+                          zorder=1)
+        
+        # Set aspect ratio
+        self.ax.set_aspect('equal')
+
+        # Store the reference line
+        self.referenceLine = h
+
+        # Collect sample points for latter use (e.g. legend)
+        self.modelMarkerSet = []
+        
+        # Set number for models outside axes
+        self.modelOutside = -1
+        
+    def biasToMarkerSize(self, biasarr):
+        '''Helper function to return marker size and sign based on input bias'''
```

Comment:
```suggestion
        """Helper function to return marker size and sign based on input bias"""
```
For consistency of docstring formatting